README.rst

```diff
@@ -63,17 +63,22 @@ Tested with all combinations of:
 API
 ===
 
-``record(file_name=None, record_name=None)``
---------------------------------------------
+``record(file_name=None, record_name=None, path=None)``
+-------------------------------------------------------
 
 Return a context manager that will be used for a single performance test.
-`file_name` is the name of the performance file to be used, and `record_name`
```

Comment:
Sorry about these but it's two backticks for code in reStructuredText. I'm fixing them all 👍